java/source/org/libreoffice/ide/eclipse/java/export/JavaExportPart.java

```diff
@@ -168,8 +170,17 @@ public void doFinish(UnoPackage pModel) {
                 writer = new FileWriter(propsFile);
 
                 Properties props = new Properties();
-                props.put("office.install.dir", new String()); //$NON-NLS-1$
-                props.put("sdk.dir", new String()); //$NON-NLS-1$
+
+                if(OOoContainer.getOOoKeys().size() > 0)
+                    props.put("office.install.dir", OOoContainer.getOOoKeys().get(0)); //$NON-NLS-1$
```

Comment:
This writes the key like "LibreOffice 5.1". But we need to write the path instead, like "/usr/lib/libreoffice/".